### DISCUSSION TITLE
Leantime V2.2.0

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#760] [Announcements]
## Leantime v2.2.0 is out 🚀🚀🚀

Here the highlights:
* **In order to run this version you must update PHP to version 8.0+.**
Support for PHP 7.4 was dropped. Leantime v2.2 will not work with PHP 7
* You can now update/add/remove columns to the kanban board (either via the three dot menue on the column or under project settinga)
* We fixed over 25 issues 
* The nasty empty-email-in-config bug is fixed
* Leantime now supports 2 ldap directory types: ActiveDirectory and OpenLdap 
* European Protugese translation was added
* We added telemetry. This will help us identify feature usage and is completely anonymous. We cannot and will not track individual users. To opt-out please head to company settings and uncheck the box.

Thank you for all your support!!!

## What's Changed
* Update class.delCanvas.php by @Jacunn in https://github.com/Leantime/leantime/pull/691
* Display certain IDs in frontend by @axelrindle in https://github.com/Leantime/leantime/pull/703
* fix misdisplayed of sprintBurndownChart by @raowang in https://github.com/Leantime/leantime/pull/708
* Prevent access to installation page after installation by @axelrindle in https://github.com/Leantime/leantime/pull/711
* Update es-ES.ini by @Dane411 in https://github.com/Leantime/leantime/pull/714
* Lossless compression of images by @phulstaert in https://github.com/Leantime/leantime/pull/730
* Update the docker start command by @phulstaert in https://github.com/Leantime/leantime/pull/731
* cleanup langlist by @phulstaert in https://github.com/Leantime/leantime/pull/736
* update dutch language by @phulstaert in https://github.com/Leantime/leantime/pull/735
* Create CODE_OF_CONDUCT.md by @marcelfolaron in https://github.com/Leantime/leantime/pull/743
* Message Translation European Portuguese by @joaobrazuna in https://github.com/Leantime/leantime/pull/745
* Bug fixes by @marcelfolaron in https://github.com/Leantime/leantime/pull/719
* Flexible status + Bug Fixes by @marcelfolaron in https://github.com/Leantime/leantime/pull/752
* Flexible status by @marcelfolaron in https://github.com/Leantime/leantime/pull/753
* Bugs by @marcelfolaron in https://github.com/Leantime/leantime/pull/755
* Misc Bugs, Clean Up & Telemetry by @marcelfolaron in https://github.com/Leantime/leantime/pull/757
* Fixing wrong method calls by @marcelfolaron in https://github.com/Leantime/leantime/pull/758
* Bugs by @marcelfolaron in https://github.com/Leantime/leantime/pull/759

## New Contributors
* @Jacunn made their first contribution in https://github.com/Leantime/leantime/pull/691
* @axelrindle made their first contribution in https://github.com/Leantime/leantime/pull/703
* @raowang made their first contribution in https://github.com/Leantime/leantime/pull/708
* @Dane411 made their first contribution in https://github.com/Leantime/leantime/pull/714
* @phulstaert made their first contribution in https://github.com/Leantime/leantime/pull/730
* @joaobrazuna made their first contribution in https://github.com/Leantime/leantime/pull/745

**Full Changelog**: https://github.com/Leantime/leantime/compare/v2.1.9...v2.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Leantime/leantime/releases/tag/v2.2.0'>Leantime V2.2.0</a>.</em>

**Comment by phulstaert** [comment]
Hi  @marcelfolaron 

Is it possible that the #730 is not in the release? All the images are still bigger.

**Reply by marcelfolaron** [reply]
Huh, I think you are right. Somehow that change was not on my local branch when I created the zip files. Let me update that real quick. Sorry about that.

**Reply by phulstaert** [reply]
No problem; I was curious about the effects on the file size of the final release and was underwhelmed a bit, so went digging a bit deeper... :-)

**Reply by marcelfolaron** [reply]
Alright, try now. I think the biggest savings are in the screenshots folder which I usually remove for the release since those are only used in the readme. I might need to move them to a CDN

**Comment by VINISHVV** [comment]
During the installation of Lean time latest version  it showing some error like below.


![image](https://user-images.githubusercontent.com/65057910/178958384-f0bc6424-23c3-4b43-86c0-6594ea6277be.png)


While checking the Lean Time folder i found that such directory (named Vendor ) is not available in the repo.

Please do the needful.

PHP Ver - 8.1.0
MYSql - 5.7.36
Os -Windows server 2012 fresh install

**Comment by marcelfolaron** [comment]
Which zip file did you use to install? The source.zip is a copy of master
and would need to be set up with composer.

I suggest using the release package called Leantime-v2.2.0.zip which would
contain the vendor folder and all other necessary installation files

**Reply by VINISHVV** [reply]
Dear @marcelfolaron - Sorry i can't find such folder in V2.2.0

![image](https://user-images.githubusercontent.com/65057910/178974892-76d4c266-c8f2-404b-a5d6-f577ff272c4a.png)

**Comment by marcelfolaron** [comment]
Please download the release package from the releases page:
https://github.com/Leantime/leantime/releases/tag/v2.1.9

**Reply by VINISHVV** [reply]
Thanks its working fine..
